src/lib/components/inbox/bids/active_bid.tsx

```diff
@@ -0,0 +1,170 @@
+import { weekdays } from 'moment';
+import * as React from "react"
+import * as Relay from "react-relay"
+import styled from "styled-components/native"
+
+import { View, Text, TouchableWithoutFeedback } from "react-native"
+import { BodyText, Subtitle, MetadataText } from "../typography"
+
+import colors from "../../../../data/colors"
+import OpaqueImageView from "../../opaque_image_view"
+
+const Container = styled.View`
+  margin: 17px 20px 0px;
+`
+
+const Content = styled.View`
+  flex: 1;
+  flex-direction: row;
+  align-items: center;
+`
+
+const ImageView = styled(OpaqueImageView) `
+  width: 58px;
+  height: 58px;
+  border-radius: 4;
+`
+
+const MetadataContainer = styled.View`
+  margin: 0 15px;
+  flex: 1;
+  flex-direction: column;
+  justify-content: center;
+`
+
+const Separator = styled.View`
+  height: 1;
+  width: 100%;
+  background-color: ${colors["gray-regular"]};
+  margin-top: 17px;
+`
+
+const StatusLabel = styled(MetadataText) `
+  color: ${props => {
+    switch (props.status) {
+      case "winning": return "rgb(14, 218, 131)"
+      case "reserve": return "rgb(241, 175, 27)"
+      case "losing": return "rgb(244, 100, 92)"
+    }
+  }};
+`
+
+type BidStatus =
+  "winning" |
+  "reserve" |
+  "losing"
+
+interface State {
+  status: BidStatus
+  statusLabel: string
+}
+
+class ActiveBid extends React.Component<any, State> {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      status: "losing"
+    }
+
+  }
+
+  componentDidMount() {
+    this.updateStatus()
+  }
+
+  updateStatus() {
+    const bid = this.props.bid
+    const leadingBidder = bid.is_leading_bidder
+    const reserveNotMet = bid.active_bid.sale_artwork.reserve_status === "reserve_not_met"
+
+    let status = "losing"
+    if (leadingBidder) {
+      status = reserveNotMet ? "reserve" : "winning"
+    }
+
+    this.setState({ status })
+  }
+
+  get statusLabel(): string {
+    switch (this.state.status) {
+      case "winning":
+      case "reserve":
+        return "Highest Bid"
+      case "losing":
+        return "Outbid"
+    }
+  }
+
+  render() {
+    const bid = this.props.bid.active_bid
+    const imageURL = bid.sale_artwork.artwork.image.url
+    const lotNumber = bid.sale_artwork.lot_number
+    const artistName = bid.sale_artwork.artwork.artist.name
+
+    const headline = `Lot ${lotNumber} · ${artistName} `
+    const subtitle = `Current Bid: ${bid.max_bid.display} `
```

Comment:
I think we want to rearrange the GraphQL fragment to use the sale artwork that's included on it, instead of the sale artwork in the `active_bid` field. The field is only present when the user is winning, but we'd like to show the current bid of the sale artwork even if they've been outbid. I took a look at [what eigen does](https://github.com/artsy/eigen/blob/1a9878316ed534fa5e4da4e8f8c14c0ab1ab63ac/Artsy/Views/Auction/LotStandingsLotView.swift#L52) and it accesses the current bid of the [sale artwork that's included in the lot standing](https://github.com/artsy/metaphysics/blob/3ef42ffd15c0029fcf6d55acd5e700792f53c93d/schema/me/lot_standing.js#L40) itself. Let me know if I can clarify anything.